src/internal/resize-handle/styles.scss

```diff
@@ -7,16 +7,13 @@
 
 .handle {
   cursor: nwse-resize;
+  @include shared.handle();
```

Comment:
Before this change, the `Handle` component did include those styles. Now it lives in a mixin and is used in the `DragHandle` and `ResizeHandle` component.